### PR TITLE
Drawing Pieces to the Board

### PR DESCRIPTION
**Pull request by SethTeck**
I know I can clean up some of the code that deals with populating the board with piece images. Jon gave me some good info on how to do it, but I got a bit confused and decided to do it in a kind of hacky way for now. 

By the way, Thank you Jon for all your help on this. You're Awesome :+1:

### CODE REVIEW

**Comment by SethTeck** [comment]
Any thoughts on what caused this to fail? I tried logging into code ship to get more details but my CodeShip account isn't linked to this project.

**Comment by jonmast** [comment]
The reason the build is failing is because we switched the king and queen positions. You should have gotten an invite to your email for codeship. You can run rspec on your machine to see the error as well. It is a good practice to run rspec and rubocop before committing to make sure it is passing.

File: app/helpers/games_helper.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        elsif current_piece.type == 'King' && current_piece.color == 'black'
+          image_tag 'dark_king.png'
+        elsif current_piece.type == 'Queen' && current_piece.color == 'white'
+          image_tag 'light_queen.png'
+        elsif current_piece.type == 'Queen' && current_piece.color == 'black'
+          image_tag 'dark_queen.png'
+        elsif current_piece.type == 'Pawn' && current_piece.color == 'white'
+          image_tag 'light_pawn.png'
+        elsif current_piece.type == 'Pawn' && current_piece.color == 'black'
+          image_tag 'dark_pawn.png'
```
Comment (jonmast) [inline app/helpers/games_helper.rb]: Sorry I ran off and didn't help you finish this. What I had in mind was doing something like this:
`image_tag "#{current_piece.color.downcase}_#{currrent_piece.type.downcase}.png"`

**Comment by radiofreemattd** [comment]
@SethTeck - don't kid yourself. This was hard and you managed to get your head around some pretty complex stuff in a super short amount of time. Really, really nice job, man. :+1:

File: spec/features/user_visits_gamepages_spec.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+require 'rails_helper'
+
+RSpec.feature 'UserVisitsGamepages', type: :feature do
+  scenario 'successfully' do
+    game = create(:game)
+    visit game_path(game)
+
+    within('.chessboard') { expect(page).to have_css 'img', count: 32 }
```
Comment (mscoutermarsh) [inline spec/features/user_visits_gamepages_spec.rb]: love it

File: spec/controllers/games_controller_spec.rb [hunk 3]
```diff
@@ -12,7 +12,8 @@
 
   describe 'GET games#show' do
     it 'responds successfully with an HTTP 200 status code' do
-      get :show
+      game = create(:game)
+      get :show, id: game
```
Comment (mscoutermarsh) [inline spec/controllers/games_controller_spec.rb]: :+1:

**Pull request merged**